Purchases/RCHTTPStatusCodes.h

```diff
@@ -0,0 +1,13 @@
+//
+// Created by Andrés Boedo on 5/6/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, RCHTTPStatusCodes) {
+    RC_NETWORK_CONNECT_TIMEOUT_ERROR = 599
```

Comment:
I only extracted this code now, but the idea is to move all of the codes in `RCBackend` to this class as well.